src/mod_webdav.c

```diff
@@ -256,8 +255,10 @@ int mod_webdav_plugin_init(plugin *p) {
 #define WEBDAV_FLAG_MOVE_XDEV    0x10
 #define WEBDAV_FLAG_COPY_XDEV    0x20
 
-#define webdav_xmlstrcmp_fixed(s, fixed) \
-        memcmp((s), (fixed), sizeof(fixed))
+#define webdav_xmlnode_isdav(prop, attr) \
+        ((prop)->ns && \
+            0 == memcmp((prop)->ns->href, "DAV:", sizeof("DAV:")) && \
+            0 == memcmp((prop)->name, (attr), sizeof(attr)))
```

Comment:
If considering this for the future, put the memcmp() of prop->name first, to short-circuit the checks early if not a match.